docs/components/popover.md

```diff
@@ -11,6 +11,7 @@ For example a button:
 <KPop title="Cool header">
   <KButton>button</KButton>
   <template v-slot:content>
+  <KButton>text</KButton>
```

Comment:
```suggestion
```

Reply:
There's an extra button in this example that wasn't there in the original:

**Original**
![Screen Shot 2021-10-04 at 8 04 36 AM](https://user-images.githubusercontent.com/2080476/135875766-dd689ef6-00ad-499f-81a6-4690a41d0be4.png)

**New**
![Screen Shot 2021-10-04 at 8 02 56 AM](https://user-images.githubusercontent.com/2080476/135875806-d7089d84-4cc4-42b3-ad85-e7ce2ec63cbe.png)

Reply:
Yep, this was left in from me testing keyboard nav. Removed now!